work-item-comments_test.go

```diff
@@ -34,6 +35,8 @@ type TestCommentREST struct {
 }
 
 func TestRunCommentREST(t *testing.T) {
+	os.Setenv("ALMIGHTY_CONFIG_FILE_PATH", "../config.yaml")
```

Comment:
I'm not sure this makes sense because the test lives in the root directory and there is no `../`.